src/components/EditorPage.js

```diff
@@ -108,6 +138,29 @@ class EditorPage extends Component {
             <i className='ion-android-arrow-back' style={{ marginRight: 0 }} />
           </Button>
 
+          <span className={cx('EditorPage-bar-item', {
+            editable: true,
+            hover: mouseOverName,
+            edit: editName,
```

Comment:
That makes sense! I was just copying from Tile.js and wasn't sure if this stuff did anything extra. Seems to be the same way with `edit` there as well!